README.md

```diff
@@ -79,24 +99,38 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
-
+app.get("/", function(req, res){
+  console.log("Shows the app-welcome page")
+  res.send("app-welcome");
+});
+
+app.post("/", function(req, res){
+  res.send("Create");
+});
+
+app.put("/:id", function(req, res){
+  res.send("Update");
+});
+
+app.delete("/:id", function(req, res){
+  res.send("Destroy");
+});
 ```
 
-```js
-// Your answer...
-```
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
-```text
+```
+Rails is a framework build on ruby that follows strict convention in terms of the syntax and organizational structure that builds the app. Rails starts with a good amount of code that people can build off of. Express on the other hand practically starts from scratch and it is up to the developer to build the routing system and links. Also, CRUD functionality is implemented differently in both. That is, Rails uses migrations whereas Express installs node modules.
```

Comment:
:+1:  
The node modules in Express correspond to gems in Ruby/Rails.  These are the dependencies.